learningequality/studio
# Make latest webpack dev setup work with docker dev setup

## Description
This allows the latest webpack dev setup to work when developing with `docker-compose`.

## Steps to Test
- Run `make dcup`.
- After the app container has loaded, check to ensure that page css is loading.

##
I have tested this and can verify that it works.
